synaptic_reconstruction/morphology.py

```diff
@@ -71,19 +71,63 @@ def compute_radii(
 
 # TODO adjust the surface for open vs. closed structures
 def compute_object_morphology(object_, structure_name, resolution=None):
-    verts, faces, normals, _ = marching_cubes(object_, spacing=(1.0, 1.0, 1.0) if resolution is None else resolution)
+    """
+    Compute the morphology (volume and surface area) of a 2D or 3D object.
+
+    Args:
+        object_ (np.ndarray): 2D or 3D binary object array.
+        structure_name (str): Name of the structure being analyzed.
+        resolution (tuple): Physical spacing between pixels/voxels (e.g., nm or μm).
```

Comment:
We should be specific: `Physical spacing in nm`, as this is what's returned in the dataframe names later.